src/main/java/com/github/nutomic/controldlna/gui/RouteFragment.java

```diff
@@ -644,7 +644,7 @@ public void clearPlaylist()
 	private String generateTimeString(int time) {
 		int seconds = time % 60;
 		int minutes = time / 60;
-		if (minutes > 99) {
+		if (minutes > 999) {
 			return "99:99";
```

Comment:
I'm slightly concerned how three digit minutes will display on a small screen, there isn't much room already in portrait mode. If I am convinced it should be merged then 99:99 should change to 999:99

Reply:
Good point, I changed the progress bar skip increment to 5% of song length.